### PR TITLE
Add "geolocate" button to Leaflet control container

### PR DESCRIPTION
**Pull request by RickMohr**

### CODE REVIEW

File: src/nyc_trees/js/src/map.js [hunk 1]
```diff
@@ -3,6 +3,10 @@
 var $ = require('jquery'),
     L = require('leaflet');
 
+var zoom = {
+    NEIGHBORHOOD: 16
```
Comment (jwalgran) [inline src/nyc_trees/js/src/map.js]: I like having zoom levels with proper names.

**Comment by jwalgran** [comment]
The map successfully found my location and centered the map on Mesa, AZ. I think we should take a page from OTM and not zoom to the user's location if they are outside NYC.

**Comment by jwalgran** [comment]
Definitely ok to open an issue for restricting the geolocation to a bounding box rather than address it now.

**Comment by jwalgran** [comment]
On Chrome, Safari, and Firefox the "minus" zoom button is offscreen, and the entire toolbar is only visible if I maximize my browser window.

screen shot 2015-01-05 at 2 25 43 pm

**Comment by RickMohr** [comment]
@jwalgran, I don't see this problem in Chrome or Firefox on Windows.

Did you scroll all the way down?

**Comment by jwalgran** [comment]
Scrolling isn't really possible because the slippy map fills the entire window

**Comment by RickMohr** [comment]
The slippy map is currently `height:800px` pending css work. For me in Firefox that means its bottom including the minus button is scrolled off the bottom and there's a vertical scrollbar.

**Comment by jwalgran** [comment]
+1

OS X like to hide scrollbars to keep the window clutter free. Sometimes they are too hidden.

**Pull request merged**